tsconfig.json

```diff
@@ -1,17 +1,13 @@
 {
   "compilerOptions": {
-    "declaration": true,
     "esModuleInterop": true,
     "inlineSources": true,
```

Comment:
Since this is set to `noEmit`, do these two sourcemap settings do anything? If so maybe we should disable them.

Reply:
You're right. Changed in [375755d](https://github.com/MetaMask/metamask-module-template/pull/102/commits/375755dce94bb1d63c138c7b433eb2a4df68eb95).